engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/number/integer/AbsNode.java

```diff
@@ -1,32 +1,44 @@
-package org.enso.interpreter.node.expression.builtin.number.smallInteger;
+package org.enso.interpreter.node.expression.builtin.number.integer;
 
+import com.oracle.truffle.api.dsl.Fallback;
 import com.oracle.truffle.api.dsl.Specialization;
 import com.oracle.truffle.api.nodes.Node;
 import org.enso.interpreter.dsl.BuiltinMethod;
 import org.enso.interpreter.node.expression.builtin.number.utils.BigIntegerOps;
 import org.enso.interpreter.node.expression.builtin.number.utils.ToEnsoNumberNode;
+import org.enso.interpreter.runtime.number.EnsoBigInteger;
 
-@BuiltinMethod(type = "Small_Integer", name = "abs", description = "Absolute value of a number")
+@BuiltinMethod(type = "Integer", name = "abs", description = "Absolute value of a number")
 public abstract class AbsNode extends Node {
   private @Child ToEnsoNumberNode toEnsoNumberNode = ToEnsoNumberNode.create();
```

Comment:
We should probably make `ToEnsoNumberNode` inlineable. Otherwise all integer nodes will have two nodes - the main one and the `ToEnsoNumberNode` one.